src/ws/ws-server.js

```diff
@@ -9,6 +9,8 @@ const config = require('../../config');
 const Server = WebSocketClient.Server;
 const phoenix = createPhoenix(WebSocketClient, { uri: config.get('ARNAUX_URL'), timeout: 500 });
 
+const UserService = require('../services/user-service');
```

Comment:
I strongly believe that ws-server will become a separate service in (near) future. So please pass the dependency into `createWsServer` (similar to `parseCookie` and `loadProfiles`)